go/vt/mysqlctl/capabilityset.go

```diff
@@ -51,18 +52,6 @@ func (c *capabilitySet) hasMaria104InstallDb() bool {
 	return c.isMariaDB() && c.version.atLeast(ServerVersion{Major: 10, Minor: 4, Patch: 0})
 }
 
-// hasDisableRedoLog tells you if the version of MySQL in use can disable redo logging.
-//
-// As of MySQL 8.0.21, you can disable redo logging using the ALTER INSTANCE
-// DISABLE INNODB REDO_LOG statement. This functionality is intended for
-// loading data into a new MySQL instance. Disabling redo logging speeds up
-// data loading by avoiding redo log writes and doublewrite buffering.
-//
-//	https://dev.mysql.com/doc/refman/8.0/en/innodb-redo-log.html#innodb-disable-redo-logging
-func (c *capabilitySet) hasDisableRedoLog() bool {
-	return c.isMySQLLike() && c.version.atLeast(ServerVersion{Major: 8, Minor: 0, Patch: 21})
-}
-
```

Comment:
I have removed this function since nothing is using atm. I'd like to reduce anything that checks capabilities to as little as possible, since this PR also further restricts that these capabilities can only be used properly inside `mysqlctl` and with less surface, we have less potential issues where it can be used wrong.